spec/defines/nodejs_instance_spec.rb

```diff
@@ -196,4 +196,86 @@
       it { should compile.and_raise_error(/Can't remove the instance/) }
     end
   end
+
+  describe 'nodejs download url' do
+    let(:params) {{
+      :version              => 'v6.0.0',
+      :ensure               => 'present',
+      :make_install         => false,
+      :target_dir           => '/usr/local/bin',
+      :cpu_cores            => 1,
+      :default_node_version => 'v5.6.0',
+    }}
+
+    describe 'os' do
+     describe 'darwin' do
+      let(:facts) {{
+        :hardwaremodel => '2.6.32-504.12.2.el6.x86_64',
+        :kernel        => 'darwin',
```

Comment:
please add `:processorcount => 1` as this fact isn't registered in the test case and it breaks the `strict vars` build on travis